public/directives/luceneQueries.html

```diff
@@ -5,15 +5,15 @@
   <ul>
     <li ng-repeat="query in queries" class="input-list">
       <input type="text" ng-model="query.name" placeholder="Display Name">
-      <input type="text" ng-model="query.query" placeholder="Query">
+      <input ng-if="addQuery" type="text" ng-model="query.query" placeholder="Query">
       <div class="btn-group">
         <button class="btn btn-xs btn-danger" ng-click="remove($index)">
           <i class="fa fa-times"></i>
         </button>
       </div>
     </li>
   </ul>
-  <button class="btn btn-xs btn-success" ng-click="add()">
+  <button ng-if="addQuery" class="btn btn-xs btn-success" ng-click="add()">
```

Comment:
For GeoSpatial filters, I would recommend keeping the button but disabling it. Then, move the message, "Filters can be added by capturing them from the dashboard", here. That way the UI is the same as non GeoSpatial filters and conveys to the user how to add a filter.